packages/browser-sdk/src/lib/react/useRoomConnection/index.ts

```diff
@@ -105,6 +107,14 @@ export function useRoomConnection(
         },
         [dispatch],
     );
+    const spotlightParticipant = React.useCallback(
+        (clientId: string) => dispatch(doSpotlightParticipant({ id: clientId })),
```

Comment:
Should be `participantId` really to match the other actions here (same for `removeSpotlight`. This makes no different however to the operation of the API.